app/assets/javascripts/helpers/post_photo_uploader.es6

```diff
@@ -53,6 +54,10 @@ Diaspora.PostPhotoUploader = class {
           /* eslint-enable camelcase */
         }
       },
+      paste: {
+        targetElement: document.getElementById("status_message_text"),
+        promptForName: true
```

Comment:
Just a little thing: I looked what this option does, and it should open a prompt for asking for a filename. It neither opens a prompt nor do I think we need a prompt. The filename doesn't matter, so I suggest to remove this option.

Reply:
IIRC with that field set to false the request didn't percolate up correctly to initiate our handlers it just pasted into the text field like before.  I can double check that that was the case.  If it is there is nothing I can do about it even though it is confusing.

Reply:
I confirmed what I stated above.